include/arch/arm/aarch64/timer.h

```diff
@@ -20,7 +20,8 @@ extern "C" {
 #define ARM_ARCH_TIMER_PRIO	ARM_TIMER_VIRTUAL_PRIO
 #define ARM_ARCH_TIMER_FLAGS	ARM_TIMER_VIRTUAL_FLAGS
 
-#define CNTV_CTL_ENABLE		((1) << 0)
+#define CNTV_CTL_ENABLE		BIT(0)
+#define CNTV_CTL_IT_MASK	BIT(1)
```

Comment:
Missing tab

Reply:
hmm. VIM shows they are in same col. I'll check more.